### ISSUE TITLE
Build Server Automation

### ISSUE THREAD

**Issue by shiftkey** [Squirrel/OldSquirrelForWindows#59]
Given I've been targeting the VS experience with 0.7
- install
- create installer and package
- deploy manually

How should we target scenarios with automating the generation of an installer on the build server? Separate PS scripts? Another NuGet Package? Something else?

**Comment by distantcam** [comment]
MSBuild Task? It may suck but msbuild is everywhere that projects are.

Also PS isn't supported on mono.

**Comment by anaisbetts** [comment]
> Also PS isn't supported on mono.

Neither are the concept of running an installer to install programs :)   Shimmer is a project that is interesting solely on Windows.

As to the CI build, it's a bit weird - I explicitly designed this _not_ to create a Setup.exe every time you build (i.e. the act of creating a release is an Explicit action).

**Comment by distantcam** [comment]
> Neither are the concept of running an installer to install programs :)

Doh! :speak_no_evil:

It wouldn't have to create a setup.exe every time you build locally, but it would be nice to have your CI spit out an installer for you.

**Comment by anaisbetts** [comment]
> It wouldn't have to create a setup.exe every time you build locally, but it would be nice to have your CI spit out an installer for you.

Agreed - trying to think of the most CI friendly way to do that. The CI servers we use at GitHub require you to write a script anyways, so invoking the `Create-Release` script isn't a big deal there. More Windows-friendly ones that want everything to be in MSBuild might make that trickier
